### DISCUSSION TITLE
[GSOC 24] Reaching out to mentors for Project - 2. Inference of Neural model in Node.js environment using OpenVINO

### DISCUSSION THREAD

**Discussion by qxprakash** [openvinotoolkit/openvino#23664] [Google Summer of Code]
Hi @almilosz and @vishniakov-nikolai,

I'm Prakash, and I'd like to express my gratitude for the guidance and feedback you, along with @pavel-esir, @mlukasze, and @p-wysocki, have provided on my PRs and issues , It has been a great learning experience , I have contributed to GFIs in openvino and openvino.genai repo so far.

The ability to run inference on optimized DL models for my Intel UHD integrated graphics using Node.js API ,  is incredibly exciting to me. I am very much interested in the project , in light of this I would like to inquire that on top of [creating new samples for Node.js API](https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code#2-inference-of-neural-model-in-nodejs-environment-using-openvino) ,  is there any scope for me to contribute towards extending the Node.js API itself as part of my GSoC project. I will be happy to allocate additional time beyond what is described in the project description to achieve these supplemental objectives. 

I'm eager to hear your thoughts on this proposal and am happy to discuss it further.

Best regards,
Prakash

**Comment by vishniakov-nikolai** [comment]
Hi Prakash,

Thanks for contributions that you've made and for your willing to participate in GSoC! 
You successfully passed prerequisite task and now you have to submit your request using GSoC webapp: https://summerofcode.withgoogle.com/

Please, let us know if you have any question about application. 
Also notice, that the deadline for the final submission is 2nd April 6pm UTC.

BR,
Nikolai

**Reply by qxprakash** [reply]
Thanks @vishniakov-nikolai @almilosz   for your guidance and feedback , I have submitted my proposal , can you please take a look , I had few points on which I wanted to have a discussion with you , I have mailed you my proposal asking for feedback , If you're available on discord we can connect there . my discord username is **@qxprakash** I will share my doubt points with @almilosz  as well I am already connected with her on discord.

Best Regards,
Prakash
